CHANGELOG.md

```diff
@@ -1,21 +1,19 @@
 ## 2.1.0 (April 2, 2021)
-
 ### ares-device
 * Supports screen capture using the `--capture-screen` option.
 
 ### ares-generate
-* Updated the template of appinfo.json to apply new ACG groups.
+* Updated the template of appinfo.json to apply new ACG groups. Template files generated by CLI v2.0.3 or older are not compatible with the latest webOS OSE & Auto.
```

Comment:
I think I should write the content based on the currently updated content. what do you think?

Updated the template of appinfo.json to apply new ACG groups. Following this change, templates generated with CLI v2.10.0 are not compatible with previous webOS OSE and Auto.

Reply:
@ssuminahn , your suggested sentence is needed to check ACG group in previous webOS & Auto version.
I think that  "Template files generated by CLI v2.0.3 or older are not compatible with the latest webOS OSE & Auto." is clear and correct. I prefer this description.